test/controllers/content_items_controller_test.rb

```diff
@@ -72,6 +72,41 @@ class ContentItemsControllerTest < ActionController::TestCase
     assert_redirected_to content_item['base_path']
   end
 
+  test "returns HTML when an unspecific accepts header is requested (eg by IE8 and below)" do
+    request.headers["Accept"] = "*/*"
+    content_item = content_store_has_schema_example('travel_advice', 'full-country')
+
+    get :show, params: {
+      path: path_for(content_item)
+    }
+
+    assert_match(/text\/html/, response.headers['Content-Type'])
+    assert_response :success
+    assert_select '#wrapper'
+  end
+
+  test "returns a 406 for XMLHttpRequests" do
```

Comment:
I think the only reason this is ok at the moment is because we don't have CORS headers that allow use of our HTML content from other sites anyway, and we know we're not (currently) loading content via Ajax. Otherwise I would think that GETting HTML in a Javascript context would be acceptable. 

I'm not sure what to think about this.